lib/new_relic/enabled_supervisor.ex

```diff
@@ -7,7 +7,6 @@ defmodule NewRelic.EnabledSupervisor do
   @moduledoc false
 
   def start_link() do
-    NewRelic.Harvest.Collector.AgentRun.ensure_init()
```

Comment:
Instead of blocking here, we rely on getting a message back inside `EnabledSupervisorManager` before starting this supervisor